ui/src/components/custom-label/custom-label.tsx

```diff
@@ -0,0 +1,31 @@
+import classnames from 'classnames';
+
+export interface CustomLabelProps {
+  label: string;
+  required?: boolean;
+  className?: string;
+  style?: React.CSSProperties & { requiredColor?: string };
+}
+
+export const CustomLabel = ({
+  label,
+  required,
+  className,
+  style,
+}: CustomLabelProps): React.ReactElement => {
+  const classes = classnames('custom-label', className);
```

Comment:
Is this doing anything? I don't see any css added to this component

Reply:
Not at the moment, removed.